### PR TITLE
feat: Support exteneded interface  parameters

### PR DESCRIPTION
**Pull request by mato533**
Changed to be able to output Type definitions for extended interfaces.

Remacks:
[dts-dom](https://github.com/RyanCavanaugh/dts-dom)@v3.7.0 has the [bug](https://github.com/RyanCavanaugh/dts-dom/issues/88) that is related to print extends statements.
So avoid this bug, use [dts-dom](https://github.com/RyanCavanaugh/dts-dom)@v3.6.0.

Refs: #193

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "homepage": "https://github.com/fossamagna/google-script-dts-generator#readme",
   "dependencies": {
     "commander": "^7.0.0",
-    "dts-dom": "^3.6.0",
+    "dts-dom": "~3.6.0",
```
Comment (fossamagna) [inline package.json:18]: Please update `yarn.lock` too.

**Review by fossamagna — CHANGES_REQUESTED** [review]
Hey @mato533 thanks for the contribution! Functionally this change looks good to me, a small nits to address.

**Comment by mato533** [comment]
Thank you for review. I missed the updating the yarn.lock file. So I Fixed it.

File: package.json (hunk 1 above)
Comment (mato533) [reply package.json:18]: Thx! I have pushed it.

**Review by fossamagna — APPROVED** [review]

**Pull request merged**